private/rules/pin_dependencies.bzl

```diff
@@ -120,6 +121,9 @@ pin_dependencies = rule(
             doc = "Location of the generated lock file",
             mandatory = True,
         ),
+        "jvm_flags": attr.string(
```

Comment:
A `string_list` would better model what you're doing here, and would match how we set jvm args in the regular java rules.

Reply:
True, but is there value in parsing the `JDK_JAVA_OPTIONS` env var into a string_list only to immediately re-concatenate it so we can pass it to the `--jvm_flags` option of the java wrapper script?